addons/website_sale/static/src/website_builder/products_item_option.js

```diff
@@ -40,11 +56,13 @@ export class ProductsItemOption extends BaseOptionComponent {
     addClassToTableCells(x, y, className) {
         const table = this.tableRef.el;
 
-        const rows = table.rows;
-        for (let row = 0; row < y; row++) {
-            const cells = rows[row].cells;
-            for (let col = 0; col < x; col++) {
-                cells[col].classList.add(className);
+        const rows = table?.rows;
+        if(rows) {
+            for (let row = 0; row < y; row++) {
+                const cells = rows[row].cells;
+                for (let col = 0; col < x; col++) {
+                    cells[col].classList.add(className);
+                }
             }
         }
```

Comment:
This is perfectly fine.
Personally I do prefer adding a `return;` in these cases, to reduce indentations, something like:

```suggestion
        if(!rows) { return; }
        for (let row = 0; row < y; row++) {
            const cells = rows[row].cells;
            for (let col = 0; col < x; col++) {
                cells[col].classList.add(className);
            }
        }

```
Still, everyone has his own preference, feel free to use the approach you prefer :)

Reply:
I find it easier to read this way, thanks!